src/scheduler/pbs_sched.c

```diff
@@ -1366,12 +1365,11 @@ main(int argc, char *argv[])
 			if (p)
 				*p = '\0';
 		} else {
-			nodename = get_all_ips(log_buffer);
+			nodename = get_all_ips(host, log_buffer, sizeof(log_buffer) - 1);
 		}
 		if (!nodename) {
 			log_err(-1, "pbsd_main", log_buffer);
-			(void) sprintf(log_buffer, "Unable to determine TPP node name");
-			fprintf(stderr, "%s", log_buffer);
+			fprintf(stderr, "%s\n", "Unable to determine TPP node name");
 			return (3);
```

Comment:
In some places we are returning -1 and in other places we are returning 3.

Reply:
I changed the returns in the daemons to 1 (since i do not need any special exit code for this), and left the commands to -1 as that was what they were using in other error paths.